addons/binding/org.openhab.binding.heos/ESH-INF/thing/HeosSystem.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heos"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Thing Type -->
+    <bridge-type id="bridge">
+        <label>HEOS Bridge</label>
+        <description>The HEOS System Bridge</description>    
+        
+         <channels>
+            <channel typeId="reboot"                id="Reboot"></channel>    
+            <channel typeId="dynamicGroupHandling"  id="DynamicGroupHandling"></channel>  
+            <channel typeId="buildGroup"            id="BuildGroup"></channel> 
+            <channel typeId="playlists"             id="Playlists"></channel>
+            <channel typeId="rawCommand"            id="RawCommand"></channel>
+            <channel typeId="playUrl"               id="PlayUrl"></channel>
+             
+               
+        </channels>
+        
+          <properties>
+            <property name="vendor">Denon</property>            
+        </properties>
+        
+        <config-description>
+         <parameter name="name" type="text">
+               <label>Player Type</label>
+                <description>Player Type of the Bridge.</description>
+                <required>false</required>
```

Comment:
Is this a limited list of options?

Reply:
More ore less. All types of HEOS can be used as bridge. So it can be HEOS 1, 3, 5 and so on. As well as the AVRs... So its a long list I would say